src/Symfony/Component/Debug/ExceptionHandler.php

```diff
@@ -115,8 +114,12 @@ public function setFileLinkFormat($fileLinkFormat)
      * The latter takes precedence and any output from the former is cancelled,
      * if and only if nothing bad happens in this handling path.
      */
-    public function handle(\Exception $exception)
+    public function handle(\Throwable $exception)
```

Comment:
We could also add `: void` here now that the class is final. And we could probably add reasonable return types to other methods of the class as well.

Reply:
Look like it's gonna be done in https://github.com/symfony/symfony/pull/31996

Reply:
In #31996 we'we not added the `: void` return type, at least not actually
See [this comment](https://github.com/symfony/symfony/pull/31996#issuecomment-500894482) it might still be something to talk about